lib/compilers/dotnet.ts

```diff
@@ -532,6 +490,28 @@ class DotNetCompiler extends BaseCompiler {
 
         return result;
     }
+
+    override runExecutable(executable: string, executeParameters: ExecutableExecutionOptions, homeDir) {
+        const execOptionsCopy: ExecutableExecutionOptions = JSON.parse(
+            JSON.stringify(executeParameters),
+        ) as ExecutableExecutionOptions;
+
+        if (this.compiler.executionWrapper) {
+            execOptionsCopy.args = [...this.compiler.executionWrapperArgs, executable, ...execOptionsCopy.args];
+            executable = this.compiler.executionWrapper;
+        }
+
+        const extraConfiguration: DotnetExtraConfiguration = {
+            buildConfig: this.buildConfig,
+            clrBuildDir: this.clrBuildDir,
+            langVersion: this.langVersion,
+            targetFramework: this.targetFramework,
+            corerunPath: this.corerunPath,
+        };
+
+        const execEnv: IExecutionEnvironment = new this.executionEnvironmentClass(this.env);
```

Comment:
Should this be looked up in construction instead of each time, maybe?

Reply:
The class has state, because it needs to remember things from the packaged build, so you can't reuse it.

Reply:
Got it, thanks.